utils/jwtUtil.js

```diff
@@ -80,10 +83,81 @@ function checkTokenRecord(t) {
 } // checkTokenRecord
 
 /**
- * Verify jwt token. If verify successful, also check the token record in the
- * db (exists AND is not revoked) and load the user record from the db and add
- * to the request. (Skip the token record check if the token is the default UI
- * token.)
+ * Function to verify if a collector token is valid or not.
+  * @param  {object} req - request object
```

Comment:
@shriramshankar very minor: I think you meant the two * to have the same indentation. There is an extra space on front of this asterik.